library/functions.lisp

```diff
@@ -44,6 +46,12 @@
       (lisp :a (str item) (cl:format cl:t "~A: ~A~%" str item))
       Unit))
 
+  (declare print ((Into :a String) => :a -> :a))
+  (define (print item)
```

Comment:
might as well add `print-line` too

Reply:
How should `print-line` differ from `print`?